vehicle/mkvi/software/sensing/BUILD

```diff
@@ -0,0 +1,32 @@
+load("//bazel/tools:defs.bzl", "can_api_files")
+load("//bazel:defs.bzl", "cc_firmware")
+
+exports_files([
+    "sensing.yml",
+])
+
+cc_firmware(
+    name = "sensing",
+    srcs = [
+        "sensing.c",
+        "sensing.h",
+    ],
+    btldr = "//projects/btldr:brakelight_bspd_btldr",
+    target_compatible_with = [
+        "//bazel/constraints:atmega16m1",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":can_api",
+        "//libs/adc",
+        "//libs/gpio",
+        "//libs/timer",
+        "//projects/btldr:btldr_lib",
+    ],
+)
+
+can_api_files(
+    name = "can_api",
+    dbc = "//vehicle/mkv:mkv.dbc",
```

Comment:
MKVI DBC not mkv

Reply:
Also side note but make sure u add the sensing yaml to the mkvi dbc in `vehicle/mkvi/BUILD`

Reply:
got it! the path i'm using is in `vehicle/mkvi/build` is `"//vehicle/common/sensing:sensing.dbc"` just based on pattern matching -- not sure if that's correct